crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -42,14 +46,19 @@ pub struct RollupData {
 
 /// `SequencerBlockData` represents a sequencer block's data
 /// to be submitted to the DA layer.
-#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
 pub struct SequencerBlockData {
     pub(crate) block_hash: Hash,
```

Comment:
Do you still need all of this to be `pub(crate)`?

Reply:
no :D